src/components/Filters/Filters.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { generateSnapshots } from '@chanzuckerberg/story-utils';
+import { composeStories } from '@storybook/testing-react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import React from 'react';
+import * as stories from './Filters.stories';
+
+const { OverflowInteractive } = composeStories(stories);
+const consoleLogMock = jest.spyOn(console, 'log').mockImplementation();
```

Comment:
Removes the console logs from the stories. Will also remove once the filters logic is redone as stated in the summary.